operations/scheduler.go

```diff
@@ -315,6 +317,18 @@ func (s *Scheduler) storeOrUpdateOperation(ctx context.Context, operation, lastO
 	return nil
 }
 
+func updateTransitiveResources(ctx context.Context, storage storage.Repository, resources []*types.RelatedType, updateFunc func(obj types.Object)) error {
+	for _, trR := range resources {
+		if trR.OperationType != types.CREATE {
+			continue
+		}
+		if err := fetchAndUpdateResource(ctx, storage, trR.ID, trR.Type, updateFunc); err != nil {
```

Comment:
This could be in the if above if you change it to be `trR.OperationType == types.CREATE`

Reply:
I will leave it nested if, because otherwise the logic is hidden too far from the beginning of line